spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/mapper/ReflectionColumnMapper.scala

```diff
@@ -30,6 +29,26 @@ abstract class ReflectionColumnMapper[T : ClassTag] extends ColumnMapper[T] {
       paramName: String,
       columns: Map[String, ColumnRef]): Option[ColumnRef]
 
+  protected def annotationForFieldName(
```

Comment:
The reasoning I had was because this function was doing 'reflection'y stuff, it should belong to the class that claims to be the owning that functionality. I don't have opinions to offer on which way we should go. Happy to move it to the sub class.